monkey/infection_monkey/control.py

```diff
@@ -70,8 +71,9 @@ def send_telemetry(self, telem_category, json_data: str):
 
     def send_log(self, log):
         try:
-            telemetry = {"monkey_guid": GUID, "log": json.dumps(log)}
-            self._island_api_client.send_log(json.dumps(telemetry))
+            monkey_guid = get_agent_id()
+            log_contents = json.dumps(log)
+            self._island_api_client.send_log(monkey_guid, log_contents)
```

Comment:
Let's do this in `InfectionMonkey._send_log()` and remove `ControlClient.send_log()` `ControlClient` is being replaced piece by piece.